src/components/02_atoms/MultipleFields/MultipleFields.js

```diff
@@ -49,6 +49,15 @@ class MultipleFields extends Component {
     newItemAdded: false,
   };
 
+  componentDidMount() {
+    const { value, onChange: syncInitialStateWithRedux } = this.props;
```

Comment:
Does the restore functionality work with this? I remember this was removed because when restoring data, it didn't return the saved values: https://github.com/VD39/drupal-admin-ui/pull/2/files#diff-46728d046a5e7fd717af30e8f4f19283L49